build/docker/action.yml

```diff
@@ -63,6 +66,7 @@ runs:
         DOCKER_BUILDKIT: "1"
       with:
         context: ${{ inputs.context }}
+        build-args: ${{ inputs.build-args }}
```

Comment:
Will this work if you have multiple args as input?

Reply:
Yes. It's 1 arg per line, and `key=value` formatted:
```
build-args: |
    ARG1=value1
    ARG2=value2
```